_posts/2017-07-07-tech-behind-dta-website.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "The tech behind dta.gov.au"
+category: [blog]
+tag: 'technology'
+author: Brendan Gordon
+author-excerpt: "Brendan Gordon is a Developer at the Digital Transformation Agency."
+thumbnail: /images/blog-thumbnails/Blog-Thumb-DTA-website.png
+hero-image: /images/blog-content/Blog-Hero-DTA-website.png
+searchexcerpt: "The DTA website is an important part of our online presence. But as a small agency we have found our needs can be met with a simple static site. In this blog post we share our approach to development and the technology behind the DTA website."
+---
+
+![An illustration of a pony with a rainbow coloured mane.]({{ site.url }}{{ site.baseurl }}{{ page.hero-image }})
+
+*Caption: 'I want a pony' – a demonstration of a want, not a need.*
+
+The technical needs for our [website](https://www.dta.gov.au/) are relatively simple. We took a minimum viable product (MVP) approach to development – creating a simple static site – with a view that we would iterate and evolve the website as the DTA grows.
+
+Our site needs to be able to:
+- allow publishing of simple, informative content
+- support less than ten page templates
+- allow previewing of changes on the site before they go live
+- handle less than 1000 pages
+
+We’ve found that a simple static site meets our needs for the moment and is cheap, fast, secure, and easy to work with.
+
+WARNING: it’s about to get technical 
+
+## The tech behind the site 
+
+The DTA website is built with [Jekyll](https://jekyllrb.com/) — an open source static website generator. The site content is mostly written in [markdown](https://en.wikipedia.org/wiki/Markdown). Jekyll only runs whenever the content changes, and it generates a set of HTML, CSS and JavaScript files. These files are easily hosted on a basic web server.
```

Comment:
The capitalisation of markdown should be consistent - further down we say Markdown, and link somewhere else.